Typescript fails to compile with axios-retry

axios-retry version: v3.0.2

typescript version 2.8.1

Fails to compile, getting
```
node_modules/axios-retry/index.d.ts(27,3): error TS7020: Call signature, which lacks return-type annotation, implicitly has an 'any' return type.
```

When I hover over that line in the source code I get the same error. What is the reason? How can it be fixed? Don't mind making a PR if someone explains a bit on that

--- a/index.d.ts
+++ b/index.d.ts
@@ -34,7 +34,7 @@ export interface IAxiosRetry {
   (
     axios: axios.AxiosStatic | axios.AxiosInstance,
     axiosRetryConfig?: IAxiosRetryConfig
-  )
+  ): void
 }
 
 declare const axiosRetry: IAxiosRetry

